Cargo.lock

```diff
@@ -367,7 +367,6 @@ dependencies = [
  "tar",
  "tempfile",
  "thiserror 2.0.11",
- "time",
```

Comment:
Unfortunately it looks like this PR doesn't actually remove `time` from the dependency tree. As far as I can tell, it's [still being brought in by `pasetors`](https://github.com/brycx/pasetors/blob/27b179d050f95433aed1b53b9f62b62e6f65f75a/Cargo.toml#L84).

It looks like it's [primarily used by Cargo](https://crates.io/crates/pasetors/reverse_dependencies), so maybe they'd be open to a switch as well.

Reply:
Thanks for highlighting this.

To be clear, this wouldn't be a blocker for this PR